### ISSUE TITLE
[Android|iOS] "result" and "speechstart" events don't fire

### ISSUE THREAD

**Issue by WheelyWonka** [jamsch/expo-speech-recognition#52]
Hello, 

After following the README, I got the Android and iOS permission modals and after accepting, `result` nor `speechstart` events don't fire. The microphone is working because when I use Audio.Recording it records and read the recorded audio perfectly. 

Regarding plugin's permissions in app.json, i tried to put 

```
"plugins": [
      ...,
      "expo-speech-recognition"
],
```

and also

```
"plugins": [
      ...,
      [
            "expo-speech-recognition",
            {
                   "microphonePermission": "Allow $(PRODUCT_NAME) to use the microphone.",
                   "speechRecognitionPermission": "Allow $(PRODUCT_NAME) to use speech recognition.",
                   "androidSpeechServicePackages": ["com.google.android.googlequicksearchbox"]
            }
      ]
],
```

But it doesn't seem to make any differences.

Here's a bug reproduction repo: https://github.com/WheelyWonka/speech-bug-repo

Could you spot what I am missing please ? 

Thank you very much :)

**Comment by jamsch** [comment]
Looks like there's a breaking change in SDK 52 with the eventing. I'll work on a v1 for SDK 52.

**Comment by jamsch** [comment]
I've just released [v1.0.0](https://github.com/jamsch/expo-speech-recognition/releases/tag/v1.0.0) for Expo SDK 52. Let me know if everything's working okay.

**Comment by WheelyWonka** [comment]
Thank you very much @jamsch 1.0.0 dis now working perfectly on Android and iOS.
